lib/connection.js

```diff
@@ -257,7 +257,7 @@ Connection.prototype.bind = function(config, more) {
   buffer.addInt16(len);
   for(var i = 0; i < len; i++) {
     var val = values[i];
-    if(val === null || typeof val === "undefined") {
+    if (val == null) {
```

Comment:
I think this will cause '0', '', and other falsy values to be coerced into null used as a parameter? 🙅